src/tensortools.jl

```diff
@@ -344,8 +390,16 @@ This costs O(D^2) time, where D is the matrix dimension.
 
 See also: [`gershgorin_bounds`](@ref)
 """
-function gershgorin_discs(t::AbstractTensorMap{S, N, N}) where {S, N}
```

Comment:
Since `gershgorin_bounds(::AbstractTensorMap)` no longer calls this, I think we can actually remove the whole method. The same goes for `convert_to_matrix`.

Reply:
Yes, it might be something that could be in TensorKit as a general linalg method.